src/app/team/serviceForm/whoDoesItServe/WhoDoesItServeEdit.jsx

```diff
@@ -32,7 +34,10 @@ class WhoDoesItServe extends Component {
       serviceGroups.splice(groupIndex, 1);
     } else {
       serviceGroups.push({
-        allAges: true, minAge: defaultMinAge, maxAge: defaultMaxAge, name: groupName,
+        all_ages: true,
+        age_min: defaultMinAge,
+        age_max: defaultMaxAge,
+        name: groupName,
```

Comment:
TODO: check if these property names are what we are intending to put in the database and consistent with that. Does not appear to be related to documents required. Maybe some cross-contamination here with the ages PR.